CITATION.cff

```diff
@@ -710,6 +710,9 @@ authors:
   - given-names: Tharun
     family-names: K
     website: https://github.com/tharun634
+  - given-names: Thiti
+    family-names: Premrudeepreechacharn
+    website: https://tpremrud.github.io/
```

Comment:
@tpremrud 
I have added these details about you as a contributor.
Let me know if they should be updated.